fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/TransactionImpl.java

```diff
@@ -92,4 +92,11 @@ public Instant updateExpiry(final Duration amountToAdd) {
         return null;
     }
 
+    @Override
+    public void commitIfShortLived() {
+       if(this.isShortLived()) {
```

Comment:
Sorry to be that guy, but please add a space after `if`. Also, I think you'll need to merge in updates from master since there are merge conflicts.